docs/source/notes/libtorch_stable_abi.md

```diff
@@ -46,6 +46,111 @@ These headers are promised to be ABI stable across releases and adhere to a stro
 Unless absolutely necessary, we recommend the high-level C++ API in `torch/csrc/stable`
 which will handle all the rough edges of the C API for the user.
 
+## Migrating your kernel to the Libtorch stable ABI
+
+Kernels that wish to use the Libtorch stable ABI must be registered via `STABLE_TORCH_LIBRARY`. For kernels that are registered
+via pybind, before using the stable ABI, it would be useful to migrate to register them via `TORCH_LIBRARY`.
+
+Starting from a kernel that is registered via `TORCH_LIBRARY`, we must migrate the registration mechanism to `STABLE_TORCH_LIBRARY` in order to make the extension stable.
```

Comment:
```suggestion
If you'd like your kernel to be ABI stable with LibTorch, meaning you'd the ability to build for one version and run on another, your kernel must only use the limited stable ABI. This following section goes through some steps of migrating an existing kernel and APIs we imagine you would need to swap over.

Firstly, instead of registering kernels through `TORCH_LIBRARY`, LibTorch ABI stable kernels must be registered via `STABLE_TORCH_LIBRARY`. For kernels that are registered via `pybind`, before using the stable ABI, it would be useful to migrate to register them via `TORCH_LIBRARY`.
```